README.md

```diff
@@ -1,72 +1,6 @@
-[![Tests](https://github.com/graphprotocol/graph-cli/actions/workflows/ci.yml/badge.svg)](https://github.com/graphprotocol/graph-cli/actions/workflows/ci.yml)
-
-## The Graph Command Line Interface
-
-As of today, the command line interface supports the following commands:
-
-- `graph init` — Creates a new subgraph project from an example or an existing contract.
-- `graph create` — Registers a subgraph name with a Graph Node.
-- `graph remove` — Unregisters a subgraph name with a Graph Node.
-- `graph codegen` — Generates AssemblyScript types for smart contract ABIs and the subgraph schema.
-- `graph build` — Compiles a subgraph to WebAssembly.
-- `graph deploy` — Deploys a subgraph to a
-  [Graph Node](https://github.com/graphprotocol/graph-node).
-- `graph auth` — Stores a [Graph Node](https://github.com/graphprotocol/graph-node) access token in
-  the system's keychain.
-- `graph local` — Runs tests against a [Graph Node](https://github.com/graphprotocol/graph-node)
-  test environment (using Ganache by default).
-- `graph test` — Downloads and runs the [Matchstick](https://github.com/LimeChain/matchstick) rust
-  binary in order to test a subgraph.
-- `graph add` - Adds a new datasource to the yaml file and writes the necessary changes to other
-  files - schema.graphql, abi and mapping.
-
-## How It Works
-
-The Graph CLI takes a subgraph manifest (defaults to `subgraph.yaml`) with references to:
-
-- A GraphQL schema,
-- Smart contract ABIs, and
-- Mappings written in AssemblyScript.
-
-It compiles the mappings to WebAssembly, builds a ready-to-use version of the subgraph saved to IPFS
-or a local directory for debugging, and deploys the subgraph to a
-[Graph Node](https://github.com/graphprotocol/graph-node).
-
-## Installation
-
-The Graph CLI can be installed with `npm` or `yarn`:
-
-```sh
-# NPM
-npm install -g @graphprotocol/graph-cli
-
-# Yarn
-yarn global add @graphprotocol/graph-cli
-```
-
-### On Linux
-
-`libsecret` is used for storing access tokens, so you may need to install it before getting started.
-Use one of the following commands depending on your distribution:
-
-- Debian/Ubuntu: `sudo apt-get install libsecret-1-dev`
-- Red Hat: `sudo yum install libsecret-devel`
-- Arch Linux: `sudo pacman -S libsecret`
-
-## Getting Started
-
-The Graph CLI can be used with a local or self-hosted
-[Graph Node](https://github.com/graphprotocol/graph-node) or with the
-[Hosted Service](https://thegraph.com/explorer/). To help you get going, there are
-[quick start guides](https://thegraph.com/docs/en/developer/quick-start/) available for both.
-
-If you are ready to dive into the details of building a subgraph from scratch, there is a
-[detailed walkthrough](https://thegraph.com/docs/en/developer/create-subgraph-hosted/) for that as
-well, along with API documentation for the
-[AssemblyScript API](https://thegraph.com/docs/en/developer/assemblyscript-api/).
+Monorepo for various tools used by subgraph developers.
```

Comment:
Maybe we should list the tools that are managed in the repo + link to the package?